bld/CLMBuildNamelist.pm

```diff
@@ -2125,10 +2125,14 @@ sub setup_logic_roughness_methods {
     $log->fatal_error("$var is incorrect entry for the namelist variable z0param_method; expected Meier2022 or ZengWang2007");
   }
   my $phys = $physv->as_string();
```

Comment:
Add use_fates to the add_default call above for z0param_method

Reply:
That did the trick.  Thanks for the reminder.